aea/cli/add.py

```diff
@@ -123,23 +127,28 @@ def _add_item(click_context, item_type, item_public_id) -> None:
         sys.exit(1)
 
     # find and add protocol
-    if is_local:
+    if item_public_id in [DEFAULT_CONNECTION, DEFAULT_PROTOCOL, DEFAULT_SKILL]:
+        package_path = _find_item_in_distribution(ctx, item_type, item_public_id)
+        _copy_package_directory(
+            ctx, package_path, item_type, item_public_id.name, item_public_id.author
+        )
+    elif is_local:
         package_path = _find_item_locally(ctx, item_type, item_public_id)
         _copy_package_directory(
             ctx, package_path, item_type, item_public_id.name, item_public_id.author
         )
-        if item_type in {"connection", "skill"}:
-            configuration_file_name = _get_default_configuration_file_name_from_type(
-                item_type
-            )
-            configuration_path = package_path / configuration_file_name
-            configuration_loader = ConfigLoader.from_configuration_type(
-                ConfigurationType(item_type)
-            )
-            item_configuration = configuration_loader.load(configuration_path.open())
-            _add_protocols(click_context, item_configuration.protocols)
     else:
-        fetch_package(item_type, public_id=item_public_id, cwd=ctx.cwd)
+        package_path = fetch_package(item_type, public_id=item_public_id, cwd=ctx.cwd)
+    if item_type in {"connection", "skill"}:
```

Comment:
All packages need to load their dependencies, which your fetch_package command had missed!